profile/README.md

```diff
@@ -0,0 +1,23 @@
+<h2 align="center">JSON Schema is a declarative language that allows you to annotate and validate JSON documents</h2>
+
+[![Slack](https://img.shields.io/static/v1?label=Slack&message=@json-schema&color=yellow)](https://asyncapi.slack.com)
+[![Twitter](https://img.shields.io/static/v1?label=Twitter&message=@jsonschema&color=9cf)](https://twitter.com/jsonschema)
+[![LinkedIn](https://img.shields.io/static/v1?label=LinkedIn&message=@jsonschema&color=lightgray)](https://www.linkedin.com/company/jsonschema)
+[![YouTube](https://img.shields.io/static/v1?label=Youtube&message=@JSONSchemaOrgOfficial&color=red)](https://www.youtube.com/@JSONSchemaOrgOfficial)
+[![Google Groups](https://img.shields.io/static/v1?label=Google&nbsp;Groups&message=@json-schema&color==blueviolet)](https://groups.google.com/g/json-schema)
```

Comment:
```suggestion
```

Anyone object to removing the google group? I don't want to promote it given we are unable to admin it. All new messages are held in a queue, and I can't log in with my old work email to migrate the admin access.